### PR TITLE
feat: any api request builder integer contract

### PR DESCRIPTION
**Pull request by petia-latinova**

### CODE REVIEW

File: packages/hardhat/contracts/APIRequestBuilder.sol [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+        // Sends the request
+        return sendChainlinkRequestTo(oracle, request, fee);
+    }
+
+     /**
+     * Create a Chainlink request to retrieve API response, find the target
+     * data, then multiply by 1000000000000000000 (to remove decimal places from data).
+     */
+    function requestDataInSatoshi(string memory url, string memory path) public returns (bytes32 requestId)
```
Comment (imollov) [inline packages/hardhat/contracts/APIRequestBuilder.sol:70]: What's the purpose of this function?

**Comment by imollov** [comment]
@petia-latinova please check codecov warnings.

File: packages/hardhat/contracts/APIRequestBuilder.sol (hunk 1 above)
Comment (imollov) [reply packages/hardhat/contracts/APIRequestBuilder.sol:70]: It's not crucial to have it. You can pass `1` for `timesAmount` or add a condition in the function to omit the `times` param. 

In any case this contract is solely for demo purposes and performance is not as critical.

Comment (petia-latinova) [reply packages/hardhat/contracts/APIRequestBuilder.sol:70]: ok I'll remove it then.

**Review by imollov — APPROVED** [review]

**Pull request merged**